src/chunk.h

```diff
@@ -43,6 +43,7 @@ int ChunkNumOfSample(Chunk *chunk);
 timestamp_t ChunkGetLastTimestamp(Chunk *chunk);
 timestamp_t ChunkGetFirstTimestamp(Chunk *chunk);
 
-ChunkIterator NewChunkIterator(Chunk *chunk);
+ChunkIterator NewChunkIterator(Chunk* chunk, int initIndex);
```

Comment:
@ashtul you can't assume the chunk has an index, due to how compressed chunks works.

Reply:
@danni-m 
What compressed chunks?